### DISCUSSION TITLE
Location of calculated covariates

### DISCUSSION THREAD

**Discussion by sigmafelix** [Spatiotemporal-Exposures-and-Toxicology/NRT-AP-Model#221] [Q&A] [answered]
I keep calculated covariates in several rds files in my user directory. Do we put calculated covariates into under the input directory, for example, ./input/data/[data_element]/calculated or ./output?

**Comment by kyle-messier** [comment, marked as answer]
@sigmafelix  I think this is more relevant to what I just bothered y'all about. I think having ./input/data/[data_element] for raw data and then ./output is good as long as we can put those on the .gitignore and .Rbuildignore and not impact the package compilation.

**Reply by sigmafelix** [reply]
Sounds great. I will copy my covariate calculation results in RDS to ./output in the team directory. Both ./input/data and ./output are listed in either .gitignore and .Rbuildignore.

**Reply by sigmafelix** [reply]
Please refer to Kyle's answer for storing calculated covariates: @dzilber @eva0marques @dawranadeep @MAKassien @mitchellmanware @Sanisha003

**Reply by eva0marques** [reply]
What do you mean by calculated covariate? Is it a new raster or you calculate for monitor data?

**Reply by sigmafelix** [reply]
@eva0marques Covariates that are calculated at sites. I think converted rasters (i.e., NLCD tiffs) should be in ./input/data/nlcd

**Reply by eva0marques** [reply]
Ok, as an stdt format?

**Reply by sigmafelix** [reply]
I'd prefer storing plain data.frames at the moment as we already have functions to convert generic data.frames into stdt. Converting a joined data.frame into stdt will be efficient, I suppose.

**Reply by eva0marques** [reply]
Ok so basically .csv files with "lat", "lon", "time" columns + covariates with crs EPSG:4326.
